compute/cloud-client/src/test/java/compute/reservation/CrudOperationsReservationIT.java

```diff
@@ -111,7 +83,6 @@ public void testGetReservation()
   }
 
   @Test
-  @Order(4)
   public void testListReservation() throws IOException {
```

Comment:
assert on `RESERVATION_NAME` in line 91 (I cannot comment on untouched code lines). Asserting on prefix will pass the test in event there is reservation from a parallel tests running there.

Reply:
You are right, We can't predict reservations' order. That's why I'm testing prefix. Because the goal of the listReservations() method is to get list with reservations. Actually, we can delete this assertion and left only isNotNull() assertion. Would you recommend to delete this assertion?

Reply:
I suggest to use a known reservation name that is supposed to be there.
If we cannot do it then I prefer the validation that you suggest because both of them can be false positive but your suggestion let's us lose enforcing the order which makes testing run faster.

Reply:
As the purpose of this test is list the reservations and not give as some defined reservation, I prefer to left it as is.